ui_backend/pkg/rest/dashboard_findings_trends.go

```diff
@@ -90,26 +86,20 @@ func validateParams(params models.GetDashboardFindingsTrendsParams) error {
 	return nil
 }
 
-// createTimeSlots returns a slice of time slots (start time and end time).
-// The function split the total startTime-endTime period (from the API call) into timeSlotsCount time slots,
-// the finding trends will be reported based on the amount of active findings in each time slot.
-func createTimeSlots(params models.GetDashboardFindingsTrendsParams) []timeSlot {
-	timeSlots := make([]timeSlot, timeSlotsCount)
+// createTimeSlots returns a slice of points in time between endTime and startTime.
+// the finding trends will be reported based on the amount of active findings in each time.
+func createTimeSlots(params models.GetDashboardFindingsTrendsParams) []time.Time {
```

Comment:
I think that we should remove the slot concept also from all the naming, because the concept is changed to time points

Reply:
done 👍